apps/client/src/routes/dashboards/dashboard/dashboard-page.tsx

```diff
@@ -42,6 +42,7 @@ export function DashboardPage() {
 
       return widget;
     }),
+    name: dashboardQuery.data?.name,
```

Comment:
Should there be a default here if `dashboardQuery.data?` is undefined?

Reply:
Added empty string when `dashboardQuery.data?` is undefined.